catalog/core/catalog-core-resourcesizeplugin/pom.xml

```diff
@@ -114,8 +114,7 @@
                         <Embed-Transitive>true</Embed-Transitive>
                         <Export-Package />
                         <Import-Package>
-                            !com.hazelcast.client;resolution:=optional,
-                            *
+                            *;resolution:=optional
```

Comment:
✏️ `*;resolution:=optional` if fine to use for test bundles, but should be avoided otherwise. ref:
https://github.com/codice/ddf/blob/649558b16f7a41874b25da4aabe9bc5f3103228a/distribution/test/thirdparty/restito/pom.xml#L64-L69